src/timeout.py

```diff
@@ -69,8 +69,18 @@ def _submit_lapsed_nodes(self, lapsed_nodes, state, log=None):
         for node_id, node in lapsed_nodes.items():
             node_update = node.copy()
             node_update['state'] = state
+            if node['kind'] != 'checkout':
+                node_update['result'] = 'incomplete'
+            else:
+                node_update['result'] = 'pass'
```

Comment:
As the method `_submit_lapsed_nodes` is generic for different cases such as `holdoff`, `timeout`, and `done`, we need to add a condition here to make it specific for incomplete nodes:
```suggestion 
            if log !=  "DONE":
                if node['kind'] != 'checkout':
                    node_update['result'] = 'incomplete'
                else:
                    node_update['result'] = 'pass'
```